README.rst

```diff
@@ -98,6 +98,10 @@ __ https://github.com/psf/black/issues/1352
 But for the time being, this is where ``darker`` enters the stage.
 This tool is for those who want to do partial formatting right now.
 
+> "When you make this single reformatting commit, everything that comes after is semantic changes so your commit history is clean ...
+However, by doing that you forever expose yourself to commits that are a mix of semantic changes with stylistic changes."
+>  <cite>[Łukasz Langa](https://lukasz.langa.pl/)</cite> - creator of Black, [article](https://lukasz.langa.pl/36380f86-6d28-4a55-962e-91c2c959db7a/)
+
```

Comment:
This doesn't seem to render well on GitHub:
![image](https://user-images.githubusercontent.com/13725/157529591-d03caec4-3453-46e1-8f51-041f902fd63d.png)